tests/integration_test/history_services_test.py

```
@@ -1133,3 +1134,60 @@ def test_total_session_with_valid_from_date_and_to_date(mock_mongo):
     assert actual["data"] == {}
     assert actual["message"] is None
     assert actual["success"]
+
+def history_conversations_agent(*args, **kwargs):
+    json_data = json.load(open("tests/testing_data/history/conversation_history_agent.json"))
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)